examples/sampling_c3/sampling_generation/obj_to_drake_sdf.py

```diff
@@ -0,0 +1,113 @@
+"""Convex decomposition for sim since Drake cannot handle collisions with non-convex meshes without using the convex hull in order to avoid mismatched geometry.
+"""
+
+import os
+import trimesh
+from lxml import etree as ET
+
+def main(obj_file, output_dir=None, model_name=None, density=1000.0, resolution=100000, max_hulls=10):
+    # Normalize input paths
+    obj_file = os.path.abspath(obj_file)
+    if output_dir is None:
+        output_dir = os.path.dirname(obj_file)
+    else:
+        output_dir = os.path.abspath(output_dir)
+        os.makedirs(output_dir, exist_ok=True)
+
+    obj_base = os.path.basename(obj_file)
+    obj_name = os.path.splitext(obj_base)[0]
+    if model_name is None:
+        model_name = obj_name
+
+    # Load mesh
+    mesh = trimesh.load(obj_file, force="mesh", skip_materials=True)
+    mesh.density = density
+    mass = mesh.mass
+    inertia = mesh.moment_inertia
+
+    # Perform convex decomposition
+    vhacd_kwargs = {
+        'maxConvexHulls': max_hulls,
+        'resolution': resolution
+    }
+    convex_dicts = trimesh.decomposition.convex_decomposition(mesh, **vhacd_kwargs)
```

Comment:
When running the `generate_files` command, this line eventually leads to a `vhacdx` import error, even after doing `pip3 install -r examples/sampling_c3/sampling_generation/requirements.txt`.